### PR TITLE
[QI2-717] remove asyncio usage from qaoa circuit

### PR DESCRIPTION
**Pull request by napocornejo**
Error with this scrip was the usage of syncio

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -21,7 +21,7 @@ classifiers = [
 qi = "quantuminspire.cli.command_list:app"
 
 [tool.poetry.dependencies]
-python = "^3.8"
+python = "^3.9"
```
Comment (oschusler) [inline pyproject.toml]: Why did you update the version of python?

File: pyproject.toml [hunk 2]
```diff
@@ -42,6 +42,9 @@ pytest-mock = "^3.12.0"
 isort = "^5.11.4"
 black = "^23.11.0"
 pytest-asyncio = "^0.21.1"
+networkx = "^3.1"
+numpy = "^1.23.5"
+scipy = "^1.10.1"
```
Comment (oschusler) [inline pyproject.toml]: These packages don't have to be installed in the SDK. It's up to the user whether they want/need them

**Review by oschusler — CHANGES_REQUESTED** [review]

File: pyproject.toml (hunk 1 above)
Comment (napocornejo) [reply pyproject.toml]: Originally we had discussed with Marten to include another test, which needs libraries in this python version. Today we discussed that this seems a bit too much just to add one test. So it has been reverted.

File: pyproject.toml (hunk 2 above)
Comment (napocornejo) [reply pyproject.toml]: Will  be reverted for the same reason as above.

**Review by oschusler — APPROVED** [review]

**Pull request merged**